src/pages/signup/StepThree.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select from '@mui/material/Select';
+import Button from '@mui/material/Button';
+
+
+export default function StepThree() {
+  return (
+    <div>
+         <FormControl fullWidth>
+            <InputLabel>Gender</InputLabel>
+            <Select
+                labelId="demo-simple-select-label"
+                id="demo-simple-select"
+                value={gender}
```

Comment:
here `gender` is a variable that's not defined. Do we need this in a useState hook? Same thing with the values below in the other Select components